v2/test/integration/mpi_job_controller_test.go

```diff
@@ -312,7 +312,7 @@ func startController(ctx context.Context, kClient kubernetes.Interface, mpiClien
 	go kubeInformerFactory.Start(ctx.Done())
 	go mpiInformerFactory.Start(ctx.Done())
 
-	go ctrl.Run(1, ctx.Done())
+	go ctrl.Run(1, ctx.Done()) //nolint:errcheck
```

Comment:
Maybe, to avoid the following lint error, the only way is to disable lint.

```shell
run golangci-lint
  Running [/home/runner/golangci-lint-1.50.1-linux-amd64/golangci-lint run --out-format=github-actions --path-prefix=v2 --timeout=5m] in [/home/runner/work/mpi-operator/mpi-operator/v2] ...
  Error: Error return value of `ctrl.Run` is not checked (errcheck)
```